public/employers/the-sand-bar.yml

```diff
@@ -0,0 +1,30 @@
+name: The Sand Bar
+image: the-sand-bar.jpg
+officialWebsite: https://www.facebook.com/thesandbartybee/
```

Comment:
They have a site: https://pamkh9.wixsite.com/the-sand-bar1 
Granted, I think they update their Facebook page more.

Reply:
Good find. I think Facebook is updated more but the site has more information about the company, so I will switch it.